server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -2113,6 +2116,10 @@ protected HttpServerTransport newHttpTransport(NetworkModule networkModule) {
         return networkModule.getHttpServerTransportSupplier().get();
     }
 
+    protected List<NetworkPlugin.AuxTransport> newAuxTransports(NetworkModule networkModule) {
+        return networkModule.getAuxServerTransportSupplierList();
```

Comment:
There are no suppliers anymore, right?

```suggestion
        return networkModule.getAuxServerTransportsList();
```

Reply:
Thanks! Fixed this naming.